lld/COFF/LTO.cpp

```diff
@@ -87,13 +87,21 @@ lto::Config BitcodeCompiler::createConfig() {
   c.RunCSIRInstr = ctx.config.ltoCSProfileGenerate;
   c.PGOWarnMismatch = ctx.config.ltoPGOWarnMismatch;
 
-  if (ctx.config.emit == EmitKind::LLVM) {
+  switch (ctx.config.emit) {
+  case EmitKind::Obj:
```

Comment:
I don't think we need to cover the `EmitKind::Obj` case here. Maybe add `default: LLVM_UNREACHABLE(...);` instead?

Reply:
Sorry actually I meant to suggest:
```
if (ctx.config.emit == EmitKind::LLVM) {
.. 
} else if (ctx.config.emit == EmitKind::ASM) {
..
}
```
I find the switch-case is less suitable here. Forget my comment about unreachable, a potential new `EmitKind`
 enumeration might add code in other places in this function, or not all, like `EmitKind::Obj`.

Reply:
That's fine as well.